app/models/channel_queue.rb

```diff
@@ -17,6 +17,10 @@ def members_string
     "```#{formatted_response}```"
   end
 
+  def first_user_in_line
+    channel_queue_memberships.order(created_at: :asc).joins(:user).first&.user
```

Comment:
There is a NextInQueue service that's also used by the `/queue open` command, but it shouldn't be any different or more robust than this, afaik